workspaces/backend/config/default/deployment.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nbv2-backend
+  namespace: backend-system
+  labels:
+    app: nbv2-backend
+    app.kubernetes.io/component: backend
+    app.kubernetes.io/managed-by: kustomize
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 1
+  selector:
+    matchLabels:
+      app: nbv2-backend
+      app.kubernetes.io/component: backend
+      app.kubernetes.io/managed-by: kustomize
+  template:
+    metadata:
+      labels:
+        app: nbv2-backend
+        app.kubernetes.io/component: backend
+        app.kubernetes.io/managed-by: kustomize
+    spec:
+      serviceAccountName: nbv2-backend
+      securityContext:
+        runAsNonRoot: true
+      terminationGracePeriodSeconds: 30
+      containers:
+      - name: nbv2-backend
+        image: nbv2-backend
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - "ALL"
+        ports:
+        - containerPort: 4000
```

I feel like we should make this a _named port_ - so we can then refer to the `name` itself in the `livenessProbe` and `readinessProbe`

```suggestion
        - name: http-api
           containerPort: 4000
```